src/main/java/miku/task/Event.java

```diff
@@ -0,0 +1,74 @@
+package miku.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an event object.
+ */
+public class Event extends Task {
+    private String dateTime1 = "";
+    private String dateTime2 = "";
+    private LocalDateTime startDateTime;
+    private LocalDateTime endDateTime;
```

Comment:
Great job on integrating the LocalDateTime java package into your Event class as well! I didn't do it so props to you!